examples/da/plot_otda_classes.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
```

Comment:
You chose to move the da examples in a sub-folder do you think it is more clear or is it to be similar to sk-learn?  did you remove the old ones, every trace of the old class should disappear from the examples?

Reply:
- I think sub-categories would clearer for examples and later on for the doc generation
- I did not remove the old ones, shoud I move them in the new sub-folder ? @agramfort : do you have some advice to deprecate examples / documention jointly with some code ?

Reply:
I think we shoud remove them from the examples since it's what people look at when trying out the toolbox.

Reply:
Ok I do that so

Reply:
yes remove deprecated code from all examples and documentation